src/cheri_cap_common.sail

```diff
@@ -32,21 +32,105 @@
 /*  SUCH DAMAGE.                                                          */
 /*========================================================================*/
 
-
+let MAX_ADDR = MAX(cap_addr_width)
```

Comment:
We should use consistent naming. In this file we have 3 conflicting styles now: `MAX_ADDR`, `max_otype` and `maxE`.

Reply:
Agreed!

Reply:
Do you think the cap_ prefix should be used for these also?

Reply:
That probably makes sense. It also makes it less likely to conflict with any future base riscv extensions.